app/controllers/network_router_controller.rb

```diff
@@ -124,8 +73,8 @@ def create
       @router = NetworkRouter.new
       options = form_params(params)
       options.merge!(form_external_gateway(params)) if switch_to_bool(params[:external_gateway])
-      ems = ExtManagementSystem.find(options[:ems_id])
-      options.delete(:ems_id)
+
+      ems = ExtManagementSystem.find(params[:ext_management_system][:id])
```

Comment:
I'm confused about `params[:ems_id]` and `params[:ext_management_system][:id]`. As it's 
 a same thing but you need one for Edit and another for Add. Please use one for both :)

Reply:
I'll make sure to use either one consistently, thanks

Reply:
So.. after https://github.com/ManageIQ/manageiq-ui-classic/pull/2422/commits/307e6c1394f576d680dd4013182917cbae3fad2a, this change is not needed at all here, right?